### PR TITLE
Add option to disable/enable mouse window dragging animations

### PR DESCRIPTION
**Pull request by DashieTM**
#### Describe your PR, what does it fix/add?
Add a configuration flag for windows being animated when dragged with the mouse.
This doesn't replace animate_manual_resizes, instead it is an addition.

wiki entry: https://github.com/hyprwm/hyprland-wiki/pull/141

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
Users would need to update their config to keep the floating window animations.

#### Is it ready for merging, or does it need work?
ready to merge.

### CODE REVIEW

File: src/layout/IHyprLayout.cpp [hunk 1]
```diff
@@ -272,7 +272,7 @@ void IHyprLayout::onMouseMove(const Vector2D& mousePos) {
     const auto DELTA     = Vector2D(mousePos.x - m_vBeginDragXY.x, mousePos.y - m_vBeginDragXY.y);
     const auto TICKDELTA = Vector2D(mousePos.x - m_vLastDragXY.x, mousePos.y - m_vLastDragXY.y);
 
-    const auto PANIMATE = &g_pConfigManager->getConfigValuePtr("misc:animate_manual_resizes")->intValue;
+    const auto PANIMATE = &g_pConfigManager->getConfigValuePtr("misc:animate_mouse_windowdragging")->intValue;
```
Comment (vaxerski) [inline src/layout/IHyprLayout.cpp:275]: what about resizing?

Comment (DashieTM) [reply src/layout/IHyprLayout.cpp]: I really wonder what I am doing sometimes

Comment (vaxerski) [reply src/layout/IHyprLayout.cpp]: please direct those questions to your therapist and not me :)

**Comment by vaxerski** [comment]
I don't like though what this has created. Ugh. When you drag a tiled window it "snaps". Should be enabled by default TBH.

**Comment by vaxerski** [comment]
it's still IMHO a result of a poor curve choice for windowMove.

**Comment by DashieTM** [comment]
I mean I used the default curve, and it was really laggy. Just like the problem shown in the video. 
From then on I disabled the manual resize animation.
I guess just see it as an additional animation option.

**Pull request merged**